autoload/db_ui/schemas.vim

```diff
@@ -141,12 +141,31 @@ let s:oracle = {
       \   'filetype': 'plsql',
       \ }
 
+let s:bigquery_list_schema_query = "SELECT schema_name FROM INFORMATION_SCHEMA.SCHEMATA;"
+
+let s:bigquery_tables_and_views = "SELECT table_schema, table_name FROM `region-us`.INFORMATION_SCHEMA.SCHEMATA;"
```

Comment:
I'm not familiar with bigquery, but this `region-us` seems like a variable.

Reply:
Hi @kristijanhusak, that's a great question. It is a variable that different companies may set based on their data location in BigQuery. Although in practice a lot of orgs/people/companies are going to be in `region-us`, this isn't universally true.

Unfortunately to retrieve the tables metadata in the same way as other schemes in the drawer here, there has to be either:
- a `region` set to use BQ's [information schema](https://cloud.google.com/bigquery/docs/information-schema-intro)
- a `dataset` (~similar to a *database* in postgresql) set
![image](https://user-images.githubusercontent.com/10452129/222335475-0b019bcb-9af6-426e-a60e-235ef00c49c8.png)
(Please see the [docs on the information schema views offered](https://cloud.google.com/bigquery/docs/information-schema-tables) for reference).

The first option to use a `dataset` is context dependent on which dataset you want tables for (so it cannot be a static query string).

The second option requires a region variable.

Would you have any suggestions for how to incorporate this as a setting?